torch/csrc/distributed/c10d/FlightRecorder.hpp

```diff
@@ -104,8 +107,10 @@ struct FlightRecorder {
     return instance;
   }
   FlightRecorder() {
-    max_entries_ = getCvarInt({"TORCH_NCCL_TRACE_BUFFER_SIZE"}, 0);
-    capture_cpp_stack_ = getCvarBool({"TORCH_NCCL_TRACE_CPP_STACK"}, false);
+    max_entries_ = getCvarInt(
+        {"TORCH_TRACE_BUFFER_SIZE", "TORCH_NCCL_TRACE_BUFFER_SIZE"}, 0);
+    capture_cpp_stack_ = getCvarBool(
+        {"TORCH_TRACE_CPP_STACK", "TORCH_NCCL_TRACE_CPP_STACK"}, false);
```

Comment:
nit: maybe we should name this `TORCH_FR_*` or `TORCH_C10D_TRACE_.*`?

Trace is a bit overloaded since it could also be referring to profilers etc